### PR TITLE
[ui][new] Conditional flow feature

### PR DESCRIPTION
**Pull request by mcada**
//test: `@integrations-conditional-flows`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Following scenarios failed:
```
Conditional flows - content base routing | Conditional flows icon test 1 | <font color="red"><b>FAILED</b></font> | mcada@redhat.com
Conditional flows - content base routing | Conditional flows icon test 2 | <font color="red"><b>FAILED</b></font> | mcada@redhat.com
Conditional flows - content base routing | Conditional flows icon test 3 | <font color="red"><b>FAILED</b></font> | mcada@redhat.com
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Checkstyle verification failed with:
```
ui-tests/src/test/java/io/syndesis/qe/pages/integrations/fragments/
  IntegrationFlowView.java:11:8: Unused import - io.syndesis.qe.steps.CommonSteps. [UnusedImports]
ui-tests/src/test/java/io/syndesis/qe/steps/other/
  ConditionalFlowSteps.java:143: 'if' construct must use '{}'s. [NeedBraces]
ui-tests/src/test/java/io/syndesis/qe/steps/other/
  ConditionalFlowSteps.java:143:9: 'if' is not followed by whitespace. [WhitespaceAround]
ui-tests/src/test/java/io/syndesis/qe/steps/other/
  ConditionalFlowSteps.java:146: 'block' child has incorrect indentation level 16, expected level should be 12. [Indentation]
ui-tests/src/test/java/io/syndesis/qe/steps/other/
  ConditionalFlowSteps.java:151: 'block' child has incorrect indentation level 16, expected level should be 12. [Indentation]
ui-tests/src/test/java/io/syndesis/qe/steps/other/
  ConditionalFlowSteps.java:152: 'block rcurly' has incorrect indentation level 12, expected level should be 8. [Indentation]
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Checkstyle verification failed with:
```
ui-tests/src/test/java/io/syndesis/qe/pages/integrations/fragments/
  IntegrationFlowView.java:11:8: Unused import - io.syndesis.qe.steps.CommonSteps. [UnusedImports]
```

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: ui-tests/src/test/java/io/syndesis/qe/steps/other/ConditionalFlowSteps.java [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+    //  |position   |ACTION   |
+    //  |1          |UP       |
+    //  |0          |DOWN     |
+    //  |1          |DELETE   |
+    @When("^click on the condition icon$")
+    public void sendEmail(DataTable data) {
+        for (List<String> row : data.cells()) {
+            switch (row.get(1)) {
+                case "UP":
+                    $(EditFlowStepElements.getConditionIconsOnPosition(row.get(0)))
```
Comment (asmigala) [inline ui-tests/src/test/java/io/syndesis/qe/steps/other/ConditionalFlowSteps.java]: Having just the `EditFlowStepElements.ICON_FOO` assignment inside the switch statement and the rest of the call outside will make this easier to refactor/fix in the future

File: ui-tests/src/test/resources/features/integrations/conditional-flow.feature [hunk 2]
```diff
@@ -0,0 +1,418 @@
...
+    And fill in values by element data-testid
+      | contextloggingenabled | true |
+      | bodyloggingenabled    | true |
+    And click on the "Next" button
+
+    When add integration step on position "0"
+    And select the "Conditional Flows" connection
+
+
+  @integrations-conditional-flows-icon-test-1
```
Comment (asmigala) [inline ui-tests/src/test/resources/features/integrations/conditional-flow.feature]: Can we get a more descriptive tags here and in the scenario below?

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**